src/biokbase/narrative/tests/test_job.py

```diff
@@ -757,7 +821,7 @@ def test_in_cells__batch__same_cell(self):
         batch_job, child_jobs = batch_fam[0], batch_fam[1:]
 
         for job in child_jobs:
-            job.cell_id = "hello"
+            job._acc_state["job_input"]["narrative_cell_info"]["cell_id"] = "hello"
```

Comment:
Was this the only place you could find where an attribute was changed (other than via the `update_state` method)?

Reply:
I checked every field in `job.__setattr__` that was from the `"job_input"`. I didn't check anything in the outer level of the ee2 state.

Reply:
But `update_state` is the only place `_acc_state` is mutated

Reply:
you left a TODO comment about whether the attribute setter was ever used in `job.py` -- seems as though you've answered it here!